### PR TITLE
feat(icons): add IconFolderTree icon

### PR DESCRIPTION
**Pull request by tjgupta**
Adds this icon:
Screen Shot 2019-10-28 at 11 51 17 AM

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @tjgupta has signed the CLA. Thanks for the pull request!

File: src/icons/general/IconFolderTree.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+import AccessibleSVG from '../accessible-svg';
+import { bdlGray50 } from '../../styles/variables';
+import type { Icon } from '../flowTypes';
+
+const IconFolderTree = ({ className = '', color = bdlGray50, height = 16, title, width = 16 }: Icon) => (
+    <AccessibleSVG
+        className={`icon-folder-tree ${className}`}
+        height={height}
+        title={title}
+        viewBox="0 0 16 16"
```
Comment (priyajeet) [inline src/icons/general/IconFolderTree.js]: 32 x 32

File: src/icons/general/IconFolderTree.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+// @flow
+import * as React from 'react';
+
+import AccessibleSVG from '../accessible-svg';
+import { bdlGray50 } from '../../styles/variables';
+import type { Icon } from '../flowTypes';
+
+const IconFolderTree = ({ className = '', color = bdlGray50, height = 16, title, width = 16 }: Icon) => (
+    <AccessibleSVG
+        className={`icon-folder-tree ${className}`}
```
Comment (priyajeet) [inline src/icons/general/IconFolderTree.js]: bdl- SUIT style name

Comment (priyajeet) [reply src/icons/general/IconFolderTree.js]: eg: https://github.com/box/box-ui-elements/blob/master/src/icons/general/IconPuzzlePiece.js

**Pull request merged**